rpm/archivematica/package.spec

```diff
@@ -0,0 +1,340 @@
+EnvironmentFile=-/etc/sysconfig/archivematica-dashboard
+WorkingDirectory=/usr/share/archivematica/dashboard/
+ExecStart=/usr/lib/python2.7/archivematica/dashboard/bin/gunicorn --workers 2 --timeout 120 --access-logfile /var/log/archivematica/dashboard/gunicorn.log --error-logfile /var/log/archivematica/dashboard/gunicorn_error.log --log-level error --bind localhost:7600 wsgi:application
+ExecReload=/bin/kill -s HUP $MAINPID
+ExecStop=/bin/kill -s TERM $MAINPID
+PrivateTmp=true
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+mkdir -p %{buildroot}/etc/sysconfig/
+cat << EOF > %{buildroot}/etc/sysconfig/archivematica-dashboard
+DJANGO_SECRET_KEY=CHANGE_ME_WITH_A_SECRET_KEY
+DJANGO_SETTINGS_MODULE=settings.common
+ARCHIVEMATICA_DASHBOARD_DB_NAME=MCP
+ARCHIVEMATICA_DASHBOARD_DB_USER=archivematica
+ARCHIVEMATICA_DASHBOARD_DB_PASSWORD=demo
+ARCHIVEMATICA_DASHBOARD_DB_HOST=localhost
+ARCHIVEMATICA_DASHBOARD_DB_PORT=3306
+ARCHIVEMATICA_DASHBOARD_GEARMAN=localhost:4730
+ARCHIVEMATICA_DASHBOARD_ELASTICSEARCH=localhost:9200
+
+EOF
+
+mkdir -p %{buildroot}/etc/sysconfig/
+cat << EOF > %{buildroot}/etc/sysconfig/archivematica-mcp-server
+DJANGO_SECRET_KEY=CHANGE_ME_WITH_A_SECRET_KEY
+ARCHIVEMATICA_MCPSERVER_DB_NAME=MCP
+ARCHIVEMATICA_MCPSERVER_DB_USER=archivematica
+ARCHIVEMATICA_MCPSERVER_DB_PASSWORD=demo
+ARCHIVEMATICA_MCPSERVER_DB_HOST=localhost
+ARCHIVEMATICA_MCPSERVER_DB_PORT=3306
+EOF
+
+
+
+# MCP Server
+mkdir -p %{buildroot}/usr/lib/archivematica/MCPServer
+cp %{_sourcedir}/%{name}/src/MCPServer/lib/* %{buildroot}/usr/lib/archivematica/MCPServer/
+
+mkdir -p %{buildroot}/usr/share/archivematica
+cp -rf  %{_sourcedir}/%{name}/src/MCPServer/share/* %{buildroot}/usr/share/archivematica/
+
+mkdir -p %{buildroot}/etc/archivematica/MCPServer/
+cp  %{_sourcedir}/%{name}/src/MCPServer/etc/serverConfig.conf %{buildroot}/etc/archivematica/MCPServer/serverConfig.conf
+
+mkdir -p %{buildroot}/usr/lib/systemd/system/
+cat << EOF > %{buildroot}/usr/lib/systemd/system/archivematica-mcp-server.service
+[Unit]
+Description=Archivematica MCP Server
+After=syslog.target network.target
+
+[Service]
+User=archivematica
+EnvironmentFile=-/etc/sysconfig/archivematica-mcp-server
+ExecStart=/usr/lib/python2.7/archivematica/dashboard/bin/python /usr/lib/archivematica/MCPServer/archivematicaMCP.py
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+
+# MCP Client
+mkdir -p %{buildroot}/etc/archivematica/MCPClient %{buildroot}//usr/lib/archivematica/MCPClient %{buildroot}/usr/lib/python2.7/archivematica/MCPClient/
+virtualenv /usr/lib/python2.7/archivematica/MCPClient/
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install --upgrade pip 
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install -r %{_sourcedir}/%{name}/src/archivematicaCommon/requirements/base.txt
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install gearman
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install MySQL
+/usr/lib/python2.7/archivematica/MCPClient/bin/pip install django-extensions
```

Comment:
Please, send PR to archivematica.git adding gunicorn to `base.txt` and that `wsgi.py` file that you are populating now from the package!